src/js/pages/OffersList/components/SorterFilter.scss

```diff
@@ -40,7 +40,7 @@ $menu-width: 85%;
 
   &.open {
     width: $menu-width;
-    padding: 24px $filter-menu-padding-x;
+    padding: 24px;
```

Comment:
Does this make the padding too big for small screens?

Reply:
In small screens it looked okay, however at bigger screen sizes this happens:
![Screenshot](https://user-images.githubusercontent.com/1106587/57653511-fad4a300-759f-11e9-8495-878f46f8cf18.png)

Reply:
This screen capture is before or after the fix?

Reply:
before the fix

Reply:
Ah ok, I'm curious now with the fix if on small screens the padding is too big